packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js

```diff
@@ -758,21 +880,18 @@ export default class WebRTCMediaEngine {
     if (!kind) {
       throw new Error('kind is required');
     }
-    if (!this.remoteMediaStream) {
-      throw new Error('No remote media stream available');
-    }
-    const tracks = this.remoteMediaStream
-      .getTracks()
-      .filter((t) => t.kind === kind);
+    const receivers = this.pc
```

Comment:
We do this code block a lot, may be useful to create a helper function

Reply:
Yea... haha I actually added getters to the class so that the phone plugin can easily get a list of senders and receivers, but haven't gone through and updated the actually internal calls.